examples/airflow/data_preprocessing_gcp_sa.yaml

```diff
@@ -0,0 +1,33 @@
+resources:
+  cpus: 1+
+
+envs:
+  DATA_BUCKET_NAME: sky-demo-data-test
+  DATA_BUCKET_STORE_TYPE: s3
+  GCP_SERVICE_ACCOUNT_JSON_PATH: null
+
+file_mounts:
+  /data:
+    name: $DATA_BUCKET_NAME
+    store: $DATA_BUCKET_STORE_TYPE
+  /tmp/gcp-service-account.json: $GCP_SERVICE_ACCOUNT_JSON_PATH
```

Comment:
Do we also need GCP credential mounting for the train and eval example YAMLs?

Reply:
https://github.com/skypilot-org/mock-train-workflow/blob/main/eval.yaml
https://github.com/skypilot-org/mock-train-workflow/blob/main/train.yaml

Reply:
I intentionally added it only for the preprocessing task, because the changes will be exactly the same for all three example YAMLs. But we could add it too for consistency.